recipes/opencv/4.x/conanfile.py

```diff
@@ -200,14 +200,17 @@ def requirements(self):
         if self.options.with_eigen:
             self.requires("eigen/3.3.9")
```

Comment:
Could we also bump Eigen required version to 3.4.0?
```suggestion
            self.requires("eigen/3.4.0")
```